src/tests/ftest/launch.py

```diff
@@ -2772,14 +2772,15 @@ def _process_core_files(self, test_job_results, test):
             self._fail_test(self.result.tests[-1], "Process", message, sys.exc_info())
             return 256
 
-        if corefiles_processed > 0 and str(test) not in TEST_EXPECT_CORE_FILES:
-            message = "One or more core files detected after test execution"
-            self._fail_test(self.result.tests[-1], "Process", message, None)
-            return 2048
-        if corefiles_processed == 0 and str(test) in TEST_EXPECT_CORE_FILES:
-            message = "No core files detected when expected"
-            self._fail_test(self.result.tests[-1], "Process", message, None)
-            return 256
+        if not core_file_processing.is_el7():
+            if corefiles_processed > 0 and str(test) not in TEST_EXPECT_CORE_FILES:
+                message = "One or more core files detected after test execution"
+                self._fail_test(self.result.tests[-1], "Process", message, None)
+                return 2048
+            if corefiles_processed == 0 and str(test) in TEST_EXPECT_CORE_FILES:
+                message = "No core files detected when expected"
+                self._fail_test(self.result.tests[-1], "Process", message, None)
+                return 256
```

Comment:
Personally, I prefer:
```suggestion
        if core_file_processing.is_el7():
            return 0

        if corefiles_processed > 0 and str(test) not in TEST_EXPECT_CORE_FILES:
            message = "One or more core files detected after test execution"
            self._fail_test(self.result.tests[-1], "Process", message, None)
            return 2048
        if corefiles_processed == 0 and str(test) in TEST_EXPECT_CORE_FILES:
            message = "No core files detected when expected"
            self._fail_test(self.result.tests[-1], "Process", message, None)
            return 256
```

Reply:
Done